gpu_direct_rdma_access.c

```diff
@@ -870,8 +916,7 @@ void rdma_close_device(struct rdma_device *rdma_dev)
         fflush(stdout);
     }
 #endif /*PRINT_LATENCY*/
-    DEBUG_LOG("ibv_destroy_qp(%p)\n", rdma_dev->qp);
-    ret_val = ibv_destroy_qp(rdma_dev->qp);
+    ret_val = destroy_qp(rdma_dev->qp);
     if (ret_val) {
         fprintf(stderr, "Couldn't destroy QP: error %d\n", ret_val);
```

Comment:
let's move this fprintf(stderr,...) into the destroy_qp() for all use cases in this code